site/content/en/docs/manual/advanced/formats/_index.md

```diff
@@ -23,23 +23,23 @@ The table below outlines the available formats for data export in CVAT.
 | [Market-1501 1.0](format-market1501)                                                                                                | .txt          | Re-identification                                           | Triplet Loss Networks, <br>Deep ReID models, and others.                                                                                                                                | Bounding Boxes                                                                         | Specific attributes  | Not supported |
-| [MOT 1.0](format-mot)                                                                                                               | .txt          | Video Tracking, <br>Detection                               | SORT, MOT-Net, IOU Tracker, <br>and others.                                                                                                                                             | Bounding Boxes, Tracks                                                                 | Specific attributes  | Supported     |
-| [MOTS PNG 1.0](format-mots)                                                                                                         | .png<br>.txt  | Video Tracking, <br>Detection                               | SORT, MOT-Net, IOU Tracker, <br>and others.                                                                                                                                             | Bounding Boxes, Tracks, Masks                                                          | Specific attributes  | Supported     |
+| [MOT 1.0](format-mot)                                                                                                               | .txt          | Video Tracking, <br>Detection                               | SORT, MOT-Net, IOU Tracker, <br>and others.                                                                                                                                             | Bounding Boxes                                                                 | Specific attributes  | Supported     |
+| [MOTS PNG 1.0](format-mots)                                                                                                         | .png<br>.txt  | Video Tracking, <br>Detection                               | SORT, MOT-Net, IOU Tracker, <br>and others.                                                                                                                                             | Bounding Boxes, Masks                                                          | Specific attributes  | Supported     |
```

Comment:
These two formats are primarily supposed for tracks.

Reply:
And?

Reply:
And the diff shows that Tracks were removed from the description.

Reply:
We have a separated column for tracks

![image](https://github.com/user-attachments/assets/5af44c03-5bb3-4dae-b343-d78fd13bf369)

Also displaying "Tracks" in column with name: "Shapes" looks weird

Reply:
BTW, I also see a small discrepancy in the `Type` column, e.g. in `JSON` and `.xml`. It seems like it also should use a uniform format.

Reply:
Agree, but I did not try to fix everything in this pr

@mdacoca